history-api/src/main/proto/ai/lzy/v1/history/history-private-api.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package ai.lzy.v1.history;
+
+import "google/protobuf/empty.proto";
+import "ai/lzy/v1/history/history.proto";
+
+service HistoryPrivateService {
+  rpc StartWorkflow (StartWorkflowRequest) returns (google.protobuf.Empty);
+  rpc ExecuteGraph (ExecuteGraphRequest) returns (google.protobuf.Empty);
+  rpc UpdateTaskStatus (UpdateTaskStatusRequest) returns (google.protobuf.Empty);
+  rpc FinishWorkflow (FinishWorkflowRequest) returns (google.protobuf.Empty);
```

Comment:
Do we want to use an Empty here?
If we want to extend the response, we have to break compatibility